Tamilcode.json

```diff
@@ -0,0 +1,453 @@
+          "\u0BA9\u0BC1",
+          "\u0BA9\u0BC2",
+          "\u0BA9\u0BC6",
+          "\u0BA9\u0BC7",
+          "\u0BA9\u0BC8",
+          "\u0BA9\u0BCA",
+          "\u0BA9\u0BCB",
+          "\u0BA9\u0BCC"
+        ]
+      }
+    ]
+  },
+  {
+    "vagai": "aayutha_eluthu",
+    "eluthu": [
+      "\u0B83"
+    ]
+  },
+  {
+    "vagai": "vada_eluthu",
+    "eluthu": [
+      {
+        "jaseries": [
+          "\u0B9C\u0BCD",
+          "\u0B9C",
+          "\u0B9C\u0BBE",
+          "\u0B9C\u0BBF",
+          "\u0B9C\u0BC0",
+          "\u0B9C\u0BC1",
+          "\u0B9C\u0BC2",
+          "\u0B9C\u0BC6",
+          "\u0B9C\u0BC7",
+          "\u0B9C\u0BC8",
+          "\u0B9C\u0BCA",
+          "\u0B9C\u0BCB",
+          "\u0B9C\u0BCC"
+        ]
+      },
+      {
+        "saseries": [
+          "\u0BB6\u0BCD",
+          "\u0BB6",
+          "\u0BB6\u0BBE",
+          "\u0BB6\u0BBF",
+          "\u0BB6\u0BC0",
+          "\u0BB6\u0BC1",
+          "\u0BB6\u0BC2",
+          "\u0BB6\u0BC6",
+          "\u0BB6\u0BC7",
+          "\u0BB6\u0BC8",
+          "\u0BB6\u0BCA",
+          "\u0BB6\u0BCB",
+          "\u0BB6\u0BCC"
+        ]
+      },
+      {
+        "shaseries": [
+          "\u0BB7\u0BCD",
+          "\u0BB7",
+          "\u0BB7\u0BBE",
+          "\u0BB7\u0BBF",
+          "\u0BB7\u0BC0",
+          "\u0BB7\u0BC1",
+          "\u0BB7\u0BC2",
+          "\u0BB7\u0BC6",
+          "\u0BB7\u0BC7",
+          "\u0BB7\u0BC8",
+          "\u0BB7\u0BCA",
+          "\u0BB7\u0BCB",
+          "\u0BB7\u0BCC"
+        ]
+      },
+      {
+        "shaaseries": [
+          "\u0BB8\u0BCD",
+          "\u0BB8",
+          "\u0BB8\u0BBE",
+          "\u0BB8\u0BBF",
+          "\u0BB8\u0BC0",
+          "\u0BB8\u0BC1",
+          "\u0BB8\u0BC2",
+          "\u0BB8\u0BC6",
+          "\u0BB8\u0BC7",
+          "\u0BB8\u0BC8",
+          "\u0BB8\u0BCA",
+          "\u0BB8\u0BCB",
+          "\u0BB8\u0BCC"
+        ]
+      },
+      {
+        "haseries": [
+          "\u0BB9\u0BCD",
+          "\u0BB9",
+          "\u0BB9\u0BBE",
+          "\u0BB9\u0BBF",
+          "\u0BB9\u0BC0",
+          "\u0BB9\u0BC1",
+          "\u0BB9\u0BC2",
+          "\u0BB9\u0BC6",
+          "\u0BB9\u0BC7",
+          "\u0BB9\u0BC8",
+          "\u0BB9\u0BCA",
+          "\u0BB9\u0BCB",
+          "\u0BB9\u0BCC"
+        ]
+      },
+      {
+        "kshaseries": [
+          "\u0B95\u0BCD\u0BB7\u0BCD",
+          "\u0B95\u0BCD\u0BB7",
+          "\u0B95\u0BCD\u0BB7\u0BBE",
+          "\u0B95\u0BCD\u0BB7\u0BBF",
+          "\u0B95\u0BCD\u0BB7\u0BC0",
+          "\u0B95\u0BCD\u0BB7\u0BC1",
+          "\u0B95\u0BCD\u0BB7\u0BC2",
+          "\u0B95\u0BCD\u0BB7\u0BC6",
+          "\u0B95\u0BCD\u0BB7\u0BC7",
+          "\u0B95\u0BCD\u0BB7\u0BC8",
+          "\u0B95\u0BCD\u0BB7\u0BCA",
+          "\u0B95\u0BCD\u0BB7\u0BCB",
+          "\u0B95\u0BCD\u0BB7\u0BCC"
+        ]
+      },
+      {
+        "shree": "\u0BB6\u0BCD\u0BB0\u0BC0"

```

Comment:
we have to keep this content as a list like other series.

Reply:
This is the same structure as uyir mei eluthu

Reply:
list of dict